### ISSUE TITLE
request to implement new people search API

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NityaNandPandey/Linkedin-SignIn#35]
```
Please implement new search API(http://developer.linkedin.com/docs/DOC-1191) 
replacing the existing search API(http://developer.linkedin.com/docs/DOC-1005) 
because the present search API is deprecated according to the linkedin 
developer Network.


```

Original issue reported on code.google.com by `contactc...@gmail.com` on 6 Jul 2010 at 12:57

**Comment by GoogleCodeExporter** [comment]
Original comment by `nabeelmukhtar` on 6 Jul 2010 at 1:01
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This defect has been fixed in build linkedin-j-1.0-rc1.zip.
```

Original comment by `nabeelmukhtar` on 9 Aug 2010 at 10:40
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
I am unable to find the fixed implementation of new search API in the current 
build download linkedin-j-1.0.335-rc2. please suggest. The searchPeople() 
methods of com.google.code.linkedinapi.client.impl.BaseLinkedInApiClient is 
still based on the old search API
```

Original comment by `contactc...@gmail.com` on 9 Nov 2010 at 1:42
